docs/source/models/concept_models/vivarium_alzheimers/concept_model.rst

```diff
@@ -281,23 +281,32 @@ The basic plan for the design of the simulation is as follows:
 5.1 Model Runs
 ------------------------
 
+.. _2025_alzheimers_model_runs_table:
+
 .. list-table:: Model run requests
   :header-rows: 1
 
   * - Number
     - Description
     - Scenarios
-    - Directory
```

Comment:
I deleted this column to make the table less wide. @albrja Can we decide on a standard naming convention for the model directories so I don't have to include them in the table?